Source/src/core/rendering/Skybox.h

```diff
@@ -22,16 +22,30 @@ namespace Hachiko
             return cube;
         }
 
+        void BuildIBL();
+
         void DrawImGui();
 
+        void BindImageBasedLightingUniforms(Program* program) const;
+
     private:
         void SelectSkyboxTexture(TextureCube::Side cube_side);
+        
+        void GenerateDiffuseIBL();
+        void GeneratePrefilteredIBL();
+        void GenerateEnvironmentBRDF();
 
         void CreateBuffers();
         TextureCube cube;
         unsigned vao{};
         unsigned vbo{};
 
+        bool activate_IBL = false;
+        unsigned diffuseIBL_id = 0;
+        unsigned prefilteredIBL_id = 0;
+        unsigned environmentBRDF_id = 0;
+        unsigned prefilteredIBL_numLevels = 0;
```

Comment:
These should be renamed like:
```suggestion
        bool activate_ibl = false;
        unsigned diffuse_ibl_id = 0;
        unsigned prefiltered_ibl_id = 0;
        unsigned environment_brdf_id = 0;
        unsigned prefiltered_ibl_number_of_levels = 0;
```